### ISSUE TITLE
🐛 notification is not triggered on duplicate classes

### ISSUE THREAD

**Issue by ohinibla** [ohinibla/class-notification#5]
Notification is not triggered on duplicate classes.
extension only checks the first occurrence as it only checks the class names.

**Comment by mazdakb** [comment]
@ohinibla Is there an `id` on the element that you can use? The `id` must be unique per element.

https://developer.mozilla.org/en-US/docs/Web/API/Element/id

**Comment by ohinibla** [comment]
> @ohinibla Is there an `id` on the element that you can use? The `id` must be unique per element.
> 
> https://developer.mozilla.org/en-US/docs/Web/API/Element/id

not really.
as you can see for instance:
```
<div class="contentRow">
  <div>
      <span>عنوان دوره</span><span>:</span>
      <h6>ماشین‌های الکتریکی 3</h6>
  </div>
  <div>
      <span>تاریخ برگزاری</span><span>:</span>
      <h6>1400/10/07</h6>
  </div>
  </div>
```
the only distinguishing factor between two items with the same name is their date, which cooperating it would require including the date in messages, sound like a mouthful but not really! just need to get familiar with codes again 😄

**Comment by mazdakb** [comment]
@ohinibla That's right. Using the date as an indicator is, as you said, fine. It's a bit unusual, but then, their website is not designed properly!

Side note: If I wanted to create this page, I would put proper (role-based) class names and ids on the elements and would never use h6 there as that's not a section title! Google also doesn't like it elements are used in this way.
